tests/Stecman/Component/Symfony/Console/BashCompletion/CompletionHandlerTest.php

```diff
@@ -148,6 +173,10 @@ protected function createHandler($commandLine, $cursorIndex = null)
      */
     protected function getTerms($handlerOutput)
     {
+        if (null === $handlerOutput) {
```

Comment:
It's more of an inconsistency than a bug, but yeah if none of the completion attempts return a value, `CompletionHandler::filterResults` is never called and execution reaches the end of the `runCompletion` method ([here](https://github.com/stecman/symfony-console-completion/blob/fe5b9e1/src/CompletionHandler.php#L96)). `CompletionHandler::runCompletion` just needs `return '';` added at the end to handle that case.